### PR TITLE
Fix jobs not loading when refreshing a scan page

### PR DESCRIPTION
**Pull request by alesgenova**
Fixes: #934

### CODE REVIEW

File: frontend/distiller/src/features/jobs/index.ts [hunk 1]
```diff
@@ -61,31 +62,33 @@ export const getScanJobs = createAsyncThunk<
...
+  if (scan) {
+    const jobIds = scan.job_ids;
+    const jobsInState = jobIds.map((id) => jobSelector(id)(state));
+    doFetch = jobsInState.some((job) => job === undefined);
   } else {
-    // If any job is not in the store, make the API call
+    // If the scan is not in the store, the getScan request hasn't resolved yet.
+    // So just fetch the jobs regardless since it's likely we haven't fetched the
+    // relevant jobs anyway.
+    doFetch = true;
```
Comment (cjh1) [inline frontend/distiller/src/features/jobs/index.ts:83]: If we initialize `doFetch` to `true`, then we don't need the `else` block.

**Comment by cjh1** [comment]
@alesgenova Not sure if this was supposed to fix the refresh issue on scans page as well. if you navigate to another page of scans for example: https://distiller-dev.lbl.gov/4dcamera/scans?page=1 and refresh the page, the scans don't get populated.

**Comment by alesgenova** [comment]
> @alesgenova Not sure if this was supposed to fix the refresh issue on scans page as well. if you navigate to another page of scans for example: https://distiller-dev.lbl.gov/4dcamera/scans?page=1 and refresh the page, the scans don't get populated.

No, that's a separate issue I'm looking into

**Review by cjh1 — APPROVED** [review]
LGTM

**Pull request merged**